pkg/agent/flags/flag_value_string.go

```diff
@@ -0,0 +1,43 @@
+package flags
+
+type stringFlagValueOption func(*stringFlagValue)
+
+func WithDefaultString(defaultVal string) stringFlagValueOption {
+	return func(s *stringFlagValue) {
+		s.defaultVal = defaultVal
+	}
+}
+
+func WithSanitizer(sanitizer func(value string) string) stringFlagValueOption {
```

Comment:
Makes sense. Though I wonder if we should write a `WithEnum` as a precanned sanitizer. Eh, wait till we need it...